agent/containers/images/visualizers/DockerfileLive

```diff
@@ -0,0 +1,37 @@
+# To be ran from parent 'visualize' directory: podman build -t <name> -f live-metric-visualizer/Dockerfile .
```

Comment:
"To be run" ... although in this context, something like "Use this with the following command" might be more clear. Also, I assume you mean "'visualizers' directory" ... and "parent" seems superfluous and possibly confusing since the file is _in_ that directory.